Tooling-Landscape/Meeting-Material/Meeting-20191010/OSTG 20191011 tooling meeting minutes.md

```diff
@@ -0,0 +1,159 @@
+# Open Source Tooling Group F2F Meeting
+
+This document summarises the in person meeting and its outcomes from the Open Source Tooling Group on October 10th 2019.
+
+## Location
+
+European Space Operations Centre
+European Space Agency ESA/ESOC
+Robert-Bosch-Str. 5
+64293 Darmstadt, Germany
+
+Room Goethe
+
+online: circuit://circuit.siemens.com/guest?token=1c2527a2-444e-4dc0-ab68-7be8bd8efcb4
+
+## Attendees
+
+In the room, in seating order:
+
+| Person | e-mail |
+| --- | --- |
+| Arun | arun.azakhezan@siemens-heathineers.com |
+| Frances | frances.paulisch@siemens-heathineers.com |
+| Andreas | andreas.isenmann@vega.com |
+| Michael | michael.hornung@salt-solutions.de |
+| Kalle | kalle.soranko@verifa.io |
+| Ben | bmarsden@verifa.io |
+| Martin | Martin.vonWillebrand@hhpartners.fi |
+| Matthew | matthew.crawford@arm.com |
+| Jean-Christophe | jean-christophe.berton@esa.int |
+| Peter | Peter.Ellsiepen@esa.int |
+| Michael (me) | michael.c.jaeger@siemens.com |
+| Lars | lars.geyer-blaumeiser@bosch-si.com |
+| Stefan | stefan.gustafsson@sap.com |
+| Jan | jan.thielscher@eacg.de |
+| Jeffrey | jeffrey.shi@sap.com |
+
+On telco, via conf call:
+
+| Person | e-mail |
+| --- | --- |
+| Helio | helio.chissini-de-castro@bmw.de |
+| Marcel | marcel.kurzmann@bosch-si.com  |
+| Monika | monika.schnizer@ts.fujitsu.com |
+| Thomas | thomas.steenbergen@here.com |
+| Hama Kouka | kouki1.hama@toshiba.co.jp |
+| Takuya Koda | takuya.koda@toshiba.co.jp |
+| Maximilian Huber | maximilian.huber@tngtech.com |
+| Philippe Ombredanne | pom@nexb.com |
+
+# Agenda and Action Items
+
+## Agenda
+
+(see slides)
+
+* Introduction
+* Updates on Integration
+* Updates on Tools
+* Tooling Big Picture Versin 2.0
+* Use cases
+* Update from Double Open
+* Update from FINOS
+* Wrap up
+
+## Action Items
+
+News Items:
+
+| Volunteering Person | Item |
+| --- | --- |
+| Michael J. | incvite for call on component naming guidelines |
+| Michael J. | creating new picture as resulting form this session |
+| Marcel, Frances, Peter | coordinate to exchange and peer-review BPMN / use case files |
+| Philippe | check if room on Friday before FOSDEM 2020 conference (Sat, Sun) can be found to have next face 2 face meeting in Bruseels on Jan 31st |
+
+Ongoing last meeting:
+
+| Volunteering Person | Item |
+| --- | --- |
+| Michael P., Lars, Michael J. | invite Peter and Andreas to present existing material |
+| Oliver  | Review SPDX Spec for updating the data model terms |
+
+# Notes on the Meeting
+
+(raw notes from notable items from discussion)
+
+## Introduction
+
+* 0915 general introduction to the location
+* 0920 Starting introduction of every participant
+
+## Updates on Tools
+
+* 0940 Integration demo: SW360antenna and SW360
+* 1010 Integration demo: Conan, O.R.T. and SW360
```

Comment:
Suggestion O.R.T. with Conan analyzer, SW360antenna and SW360

Reply:
@blaumeiser-at-bosch that's a good idea

Reply:
done.